src/course-user/course-user.service.ts

```diff
@@ -31,6 +31,7 @@ export class CourseUserService {
   async completeCourse({ userId, courseId }: CourseUserDto): Promise<CourseUserEntity> {
     const courseUser = await this.courseUserRepository.findOne({ where: { userId, courseId } });
     courseUser.completed = true;
+    courseUser.completedAt = new Date().toISOString();
```

Comment:
why `.toISOString` here? i'd have thought we want to keep it as a date everywhere

Reply:
It's similar to the date format used by Postgres/Typeorm 

2011-10-05T14:48:00.000Z